scripts/generate/templates/post-purchase/src/index.ts

```diff
@@ -55,7 +55,7 @@ async function getRenderData(): Promise<InitialState> {
 /**
  * Entry point for the `Render` Extension Point
  *
- * Returns markup composed of Argo components.  The Render extension can
+ * Returns markup composed of remote UI components.  The Render extension can
```

Comment:
I think this should be `ui extensions components` instead of `remote ui components`. Consumer of post-purchase should not know about remote-ui